pkg/daemon/ceph/osd/volume.go

```diff
@@ -269,8 +273,16 @@ func sanitizeOSDsPerDevice(count int) string {
 	return strconv.Itoa(count)
 }
 
-func getCephVolumeSupported(context *clusterd.Context) (bool, error) {
-	_, err := context.Executor.ExecuteCommandWithOutput(false, "", cephVolumeCmd, "lvm", "prepare")
+func getCephVolumeSupported(context *clusterd.Context, pvcBacked bool) (bool, error) {
+	var args []string
+	if pvcBacked {
+		args = append(args, "lvm", "prepare") //Don't use batch command when using PVC block device
```

Comment:
Since this method would be removed, I think I'll change it and won't pass `pvcBacked` argument.  So if `pvcBacked` is true, we simple set `cvSupported` to `true` and won't call `getCephVolumeSupported` at all.